packages/toolpad-core/package.json

```diff
@@ -1,75 +1,90 @@
-  "description": "Build dashboards fast.",
   "repository": {
     "type": "git",
     "url": "https://github.com/mui/mui-toolpad.git",
     "directory": "packages/toolpad-core"
   },
+  "license": "MIT",
   "bugs": {
     "url": "https://github.com/mui/mui-toolpad/issues"
   },
-  "author": "Toolpad Team",
   "homepage": "https://github.com/mui/mui-toolpad#readme",
-  "license": "MIT",
-  "main": "./dist/index.cjs",
-  "module": "./dist/index.js",
-  "types": "./dist/index.d.ts",
-  "type": "module",
-  "exports": {
-    "./package.json": "./package.json",
-    ".": {
-      "types": "./dist/index.d.ts",
-      "import": "./dist/index.js"
-    },
-    "./*": {
-      "types": "./dist/*.d.ts",
-      "require": "./dist/*.cjs",
-      "default": "./dist/*.js"
-    }
-  },
-  "files": [
-    "dist"
-  ],
   "scripts": {
-    "build": "tsup",
-    "dev": "tsup --watch",
-    "check-types": "tsup && tsc --noEmit",
+    "build": "pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files",
+    "build:node": "node ../../scripts/build.mjs node",
+    "build:stable": "node ../../scripts/build.mjs stable",
+    "build:copy-files": "node ../../scripts/copyFiles.mjs",
+    "build:types": "tsc -b tsconfig.build.json",
+    "prebuild": "rimraf build tsconfig.build.tsbuildinfo",
+    "dev": "pnpm build:stable --watch & pnpm build:types --watch",
+    "check-types": "tsc --noEmit",
     "test": "vitest run"
   },
   "dependencies": {
-    "@emotion/cache": "11.11.0",
     "@emotion/react": "11.11.4",
     "@emotion/styled": "11.11.5",
     "@mui/icons-material": "5.15.16",
     "@mui/material": "5.15.16",
-    "@mui/material-nextjs": "5.15.11",
-    "next": "14.2.3",
     "react": "18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "18.2.0"
   },
   "devDependencies": {
+    "@babel/cli": "7.24.1",
+    "@babel/core": "7.24.4",
+    "@babel/node": "7.23.9",
+    "@babel/plugin-transform-react-constant-elements": "7.24.1",
+    "@babel/plugin-transform-runtime": "7.24.3",
+    "@babel/preset-env": "7.24.4",
+    "@babel/preset-react": "7.24.1",
+    "@babel/preset-typescript": "7.24.1",
+    "@babel/register": "7.23.7",
     "@types/react": "18.3.1",
-    "@types/react-dom": "18.3.0"
+    "@types/react-dom": "18.3.0",
+    "babel-loader": "9.1.3",
+    "babel-plugin-istanbul": "6.1.1",
+    "babel-plugin-macros": "3.1.0",
+    "babel-plugin-module-resolver": "5.0.0",
+    "babel-plugin-optimize-clsx": "2.6.2",
+    "babel-plugin-react-remove-properties": "0.3.0",
+    "babel-plugin-transform-react-remove-prop-types": "0.4.24"
   },
   "peerDependencies": {
-    "@emotion/react": "^11",
-    "@emotion/styled": "^11",
     "@mui/icons-material": "^5",
     "@mui/material": "^5",
-    "@types/react": "^18",
-    "next": "^14",
-    "react": "^18",
-    "react-dom": "^18"
+    "react": "^18"
   },
+  "sideEffects": false,
   "publishConfig": {
     "access": "public"
   },
   "engines": {
     "node": ">=18"
-  }
+  },
+  "files": [
```

Comment:
Aren't we building into a `build` folder? Like core?

Reply:
The base repo uses a `build` folder but this is how the files are listed there.